### PR TITLE
Back Button

### PR DESCRIPTION
**Pull request by Ayushi-0903**
Added back button on signup and login page.

### CODE REVIEW

File: mainapp/templates/account/signup.html [hunk 1]
```diff
@@ -22,6 +24,8 @@ <h1>{% trans "Sign Up" %}</h1>
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
         <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        <a class="indent">
+        <button class="btn btn-primary" onclick="goBack()">Back</button></a>
```
Comment (ab7289) [inline mainapp/templates/account/signup.html]: Screen Shot 2019-11-02 at 4 29 34 PM

As you can see in the screenshot above the Back button is off-center from the Sign up button. I suggest you take a look at how the 3 buttons in the index page (search, sign In, Sign Up) and take inspiration from that.  Pay special attention the the `<div>`s that the buttons are wrapped in and see how they are used to position the buttons in the page.

File: mainapp/templates/account/login.html [hunk 2]
```diff
@@ -44,7 +46,14 @@ <h1>{% trans "Sign In" %}</h1>
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
         <a class="button secondaryAction" href="{% url 'account_reset_password' %}">{% trans "Forgot Password?" %}</a>
-        <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        </div>
+        <div class=""col-lg">
+        <button class="btn btn-primary">Back</button>
```
Comment (ab7289) [inline mainapp/templates/account/login.html]: the back button doesn't do anything when I click it.

File: mainapp/templates/account/signup.html [hunk 3]
```diff
@@ -21,7 +23,14 @@ <h1>{% trans "Sign Up" %}</h1>
         {% if redirect_field_value %}
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
-        <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        </div>
+        <div class=""col-lg">
+          <button class="btn btn-primary">Back</button>
```
Comment (ab7289) [inline mainapp/templates/account/signup.html]: nothing is happening when i click this back button as well

File: mainapp/templates/account/signup.html [hunk 4]
```diff
@@ -7,6 +7,8 @@
 {% block head_title %}{% trans "Signup" %}{% endblock %}
 
 {% block content %}
+<head>
```
Comment (alldne) [inline mainapp/templates/account/signup.html]: `<head>` is added. Is it an intended change?

Comment (Ayushi-0903) [reply mainapp/templates/account/signup.html]: Removed

**Comment by ab7289** [comment]
code looks good to me. please fix the failing CI build and I will approve

**Comment by Ayushi-0903** [comment]
Hey, tests is passed. Now we should merge it.

**Comment by Ayushi-0903** [comment]
Hey, tests is pass. Now we should merge it.

**Pull request merged**